site/_wiki/Development/Configurations.md

```diff
@@ -85,4 +85,20 @@ pinpoint devices.
 
 ### Attributes
 
-Attributes are used as platform specific properties to be used within utilties or tools.
+Attributes are used as platform specific properties to be used within utilities or tools.
+
+The type of the object is `Dictionary<string, string>`, e.g. `{ "FeatureInitDelayMs": "150", "libinputoverride": "1" }`.
+
+Some example attributes include:
+
+|       Key Name       |      String Value     | Description |
+| :------------------: | :-------------------: | :---------- |
+|  `libinputoverride`  |          `1`          | *(Linux only)* Whether the generic tablet interface should be ignored by [libinput] or not. Used in [udev] rule generation, using the tablets VID and PID.
+|    `MacInterface`    | non-negative integer  | *(MacOS only)* Which interface the correct tablet endpoint exists on. This helps with "randomly incorrect" detection. See the [MacInterface PR].
+|    `WinInterface`    | regex-friendly string | *(Windows only)* Similar to `MacInterface`, but matches file system path of the device with the regex `&mi_{s}`, where `{s}` is your string.
+|      `WinUsage`      | regex-friendly string | *(Windows only)* Similar to `WinInterface`, but instead checks with the regex `&col{s}`, where `{s}` is your string.
```

Comment:
```suggestion
|    `MacInterface`    | non-negative integer  | *(MacOS only)* Specifies the USB device interface to use. For example: `0`
|    `WinInterface`    | non-negative integer (digits: 2) | *(Windows only)* Similar to `MacInterface`. String should have exactly two digits, e.g., `01`
|      `WinUsage`      | non-negative integer (digits: 2) | *(Windows only)* Specifies the HID usage collection to use. String should have exactly two digits, e.g., `01`
```

Reply:
will rewrite this to be "Int Range: 00-99" or similar